extensions/adobe/experience/offer-management/capping-constraint.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/capping-constraint",
+  "type": "object",
+  "title": "Capping Constraint",
+  "description": "Offer Capping Constraint",
```

Comment:
All these descriptions and definitions assume the reader is familiar with offers and capping - please expand to make understandable by anyone.

Reply:
I am not sure that the `description` property of a schema is the right place for an introduction to Digital Marketing in general, and to Offer Management in particular. As Pierre de Fermat noted on a similar issue “this margin is too narrow to contain.”

Reply:
Description is updated and .md file is added